edflow/explore.py

```diff
@@ -201,8 +202,9 @@ def show_example(dset, idx, config):
 
     # additional visualizations
     default_additional_visualizations = retrieve(
-        config, "edexplore/visualizations", default=dict()
+        config, "edexplore/visualizations", default={}
```

Comment:
https://stackoverflow.com/a/5790954
indicates a performance improvement for this change. For consisntency however, I would not change only a single instance.
should we change dict() to {} then everywhere at leas in this file?

Reply:
I think this does not matter. I was just fixing a bug when default was not given an empty dict.